core/controllers/collection_editor_test.py

```diff
@@ -99,6 +100,68 @@ def test_access_collection_editor_page(self):
         self.assert_can_edit(response.body)
         self.logout()
 
+
```

Comment:
Should only have 1 line between tests in the same class, and 2 lines between classes.